lib/iex/lib/iex/helpers.ex

```diff
@@ -325,9 +325,26 @@ defmodule IEx.Helpers do
 
   @doc """
   Retrieves the nth expression's value from the history.
+  Where nth is iex(nth)>
 
-  Use negative values to look up expression values relative to the current one.
-  For instance, v(-1) returns the result of the last evaluated expression.
+  It is usefull to bound expression on the history to a variable
+
+  ## Examples
+
+      iex(1)> 40 + 2
+      42
+      iex(2)> [1, 2, third | rest] = [1, 2, 3, 4, 6, 7, 8, 9]
+      [1, 2, 3, 4, 6, 7, 8, 9]
+      iex(3)> third
+      3
+      iex(4)> rest
+      [4, 6, 7, 8, 9]
+      iex(5)> v(2)
+      [1, 2, 3, 4, 6, 7, 8, 9]
+      iex(6)> v(1)
+      42
+      iex(7)> meanning_of_life = v
+      42
```

Comment:
Saying "bind" over "assign" is definitely the right choice, but my comment was not about that at all :) It was about the fact that your examples are mostly showing how "binding"/"assignment" work in IEx, without too much information on `v/1`, but this is the documentation for `v/1`, so we should focus on that in the examples.

Also, `v/1` is not useful for "binding expressions in the history to a variable", but rather for accessing expression results in the history without having to bind them to a variable.